_includes/authorizations-resource.md

```diff
@@ -0,0 +1,191 @@
+        },
+        "transaction": {
+          "id": "/psp/{{payment-instrument}}/payments/{{payment-id}}/transactions/{{transaction-id}}",
+          "created": "2016-09-14T01:01:01.01Z",
+          "updated": "2016-09-14T01:01:01.03Z",
+          "type": "Authorization",
+          "state": "Initialized",
+          "number": 1234567890,
+          "amount": 1000,
+          "vatAmount": 250,
+          "description": "Test transaction",
+          "payeeReference": "AH123456",
+          "failedReason": "",
+          "isOperational": false,
+          "operations": [
+            {
+              "method": "POST",
+              "href": "https://api.stage.payex.com/psp/{{payment-instrument}}/payments/{{payment-id}}/authorizations",
+              "rel": "create-authorization",
+              "contentType": "application/json"
+            },
+            {
+              "href": "https://api.externalintegration.payex.com/psp/{{payment-instrument}}/payments/{{payment-id}}",
+              "rel": "edit-authorization",
+              "method": "PATCH"
+            }
+          ]
+        }
+      }
+    ]
+  }
+}
+```
+
+#### Create Authorization transaction
+
+To create an `authorization` transaction, perform the `create-authorization`
+operation as returned in a previously created invoice payment.
+
+{:.code-header}
+**Request**
+
+```http
+POST /psp/{{payment-instrument}}/payments/{{payment-id}}/authorizations HTTP/1.1
+Host: api.externalintegration.payex.com
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+
+{
+  "transaction": {
+    "activity": "FinancingConsumer"
+  },
+  "consumer": {
+    "socialSecurityNumber": "socialSecurityNumber",
+    "customerNumber": "123456",
+    "name": "Olivia Nyhuus",
+    "email": "olivia.nyhuus@payex.com",
+    "msisdn": "+4798765432",
+    "ip": "127.0.0.1"
+  },
+  "legalAddress": {
+    "addressee": "Olivia Nyhuus",
+    "streetAddress": "SaltnesToppen 43",
+    "zipCode": "1642",
+    "city": "Saltnes",
+    "countryCode": "no"
+  },
+  "billingAddress": {
+    "addressee": "Olivia Nyhuus",
+    "coAddress": "Bernt Nyhuus",
+    "streetAddress": "SaltnesToppen 43",
+    "zipCode": "1642",
+    "city": "Saltnes",
+    "countryCode": "no"
+  }
+}
+```
+
+{:.table .table-striped}
+| Required | Property                        | Data type | Description                                                                                                                                                      |
+| :------- | :------------------------------ | :-------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ✔︎︎︎︎︎   | `transaction.activity`          | `string`  | `FinancingConsumer`                                                                                                                                              |
```

Comment:
Ideally, code tables should use the emoji arrows to indicate hierarchy instead of spelling out the property path in the full. Since `transaction` isn't described, it needs to be added with a row in the table that describes it.

```suggestion
| ✔︎︎︎︎︎   | └➔&nbsp;`activity`          | `string`  | `FinancingConsumer`                                                                                                                                              |
```

Reply:
I have updated this table.